src/rng/CMakeLists.txt

```diff
@@ -12,17 +12,31 @@ project( rng CXX )
 # ---------------------------------------------------------------------------- #
 # Generate config.h (only occurs when cmake is run)
 # ---------------------------------------------------------------------------- #
+# default to using C++11 features of Random123.
+set( R123_USE_CXX11 1 )
+# if the current compiler doesn't support some c++11 features, disable 
+# Random123's use of these extensions.
+set( required_features "cxx_explicit_conversions;cxx_static_assert;cxx_unrestricted_unions;cxx_long_long_type;cxx_constexpr;cxx_type_traits")
+foreach( feature ${required_features} )
+   list( FIND CXX11_FEATURE_LIST ${feature} featurefound )
+   if( ${feature} STREQUAL "-1" )
```

Comment:
We should be testing the value of `featurefound` not `features` (Thanks to Alex for spotting this bug.)